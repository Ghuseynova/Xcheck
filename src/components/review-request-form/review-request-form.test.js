import React from 'react';
import { shallow } from 'enzyme';
import ReviewRequestFormCreation from './review-request-form';

describe('render ', () => {
	it('Review request form creation', () => {
		const tree = shallow(<ReviewRequestFormCreation/>);
		expect(tree).toMatchSnapshot();
	});
});