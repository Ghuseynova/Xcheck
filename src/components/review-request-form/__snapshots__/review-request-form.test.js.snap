// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render  Review request form creation 1`] = `ShallowWrapper {}`;
